test/integration_test.py

```diff
@@ -596,14 +594,14 @@ def _get_url(self,
         return response
 
     def _get_url_unchecked(self,
-                           url: str,
+                           url: furl,
                            *,
                            retries: Optional[Union[urllib3.util.retry.Retry, bool, int]] = None,
                            redirect: bool = True,
                            preload_content: bool = True) -> urllib3.HTTPResponse:
-        log.info('GET %s ...', url)
+        log.info('GET %s ...', str(url))
```

Comment:
`str` would be redundant with `%s`. But

```suggestion
        url = str(url)
        log.info('GET %r ...', url)
```

because of https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#18string-interpolation